app/components/channel_list/header/plus_menu/index.tsx

```diff
@@ -32,7 +32,13 @@ const PlusMenuList = ({canCreateChannels, canJoinChannels}: Props) => {
     }, [intl, theme]);
 
     const createNewChannel = useCallback(async () => {
-        // To be added
+        await dismissBottomSheet();
+
+        const title = intl.formatMessage({id: 'mobile.create_channel.title', defaultMessage: 'New channel'});
+        const closeButton = await CompassIcon.getImageSource('close', 24, theme.sidebarHeaderTextColor);
+        showModal(Screens.CREATE_OR_EDIT_CHANNEL, title, {
+            closeButton,
```

Comment:
instead of passing `closeButton` as a prop you have two options:
1. (preferred) create the left button here and set it as the modal options
2. get the close button in the component were you are setting the modal buttons.